packages/cli/generators/app/templates/src/migrate.ts.ejs

```diff
@@ -0,0 +1,20 @@
+import {<%= project.applicationName %>} from './application';
+
+export async function migrate(args: string[]) {
+  const existingSchema = args.includes('--rebuild') ? 'drop' : 'alter';
+  console.log('Migrating schemas (%s existing schema)', existingSchema);
+
+  const app = new <%= project.applicationName %>();
+  await app.boot();
+  await app.migrateSchema({existingSchema});
+
+  // Connectors usually keep a pool of opened connections,
+  // this keeps the process running even after all work is done.
+  // We need to exit explicitly.
+  process.exit(0);
```

Comment:
On a side note, we probably should hook up `dataSource.disconnect` with `app.stop`.

Reply:
That's a great idea! Should we also call `dataSource.connect()` from `app.start`?

Is this something to fix first, before we change the project template?

Reply:
Hmm, migrate does not call start/stop, only boot.

Should I change this block from `process.exit` to iterate over all registered datasources and call `disconnect` on them? Maybe do that as part of `app.migrate()`?

Reply:
I'm fine to use `process.exit()` for this PR since connectors are `started/connected` on demand even `app.start` is not involved. We can revisit it after the life cycle feature is ready.